client/src/component/DOS/DefineAudit/ElectionDateForm.tsx

```diff
@@ -8,20 +8,31 @@ import { timezone } from 'corla/config';
 import corlaDate from 'corla/date';
 
 
-function defaultElectionDate(): string {
-    return moment.tz(timezone).format('YYYY-MM-DD');
+function defaultElectionDate(initDate: Date): string {
+    // not sure where this string comes from
+    if (initDate && 'Invalid Date' !== initDate.toString()) {
+        // date is formatted as utc in corlaDate.format
+        // if we don't use utc here we'll get the wrong day sometimes it appears.
+        // there may be a better way to do this.
+        return moment(initDate).tz('utc').format('YYYY-MM-DD');
```

Comment:
I'd try to avoid the timezone conversion here just in case it skips ahead a day, but I don't really understand the behavior you were seeing, so maybe we just flag this to review later...

Reply:
To review later: https://www.pivotaltracker.com/story/show/158980690